specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/guestconfiguration.json

```diff
@@ -852,6 +1013,323 @@
+            "type": "string",
+            "readOnly": true,
+            "description": "Source of approval"
+          },
+      "latestAssessmentTimeTaken": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Time taken for the latest assessment to complete on the computer"
+          },
+      "lastUpdateApplied": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Last update applied"
+          },
+      "updateAssessmentErrors": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Errors that occurred during assessment, if any"
+          },
+      "updateAssessmentErrorStackTrace": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Stack Trace for any terminal error during assessment, if any"
+          },
+      "lastRefreshDate": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Last refresh date"
+          }
+        }
+      },
+    "SoftwareUpdatesList": {
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/SoftwareUpdate"
+            },
+            "description": "The collection of software updates found on the computer."
+          }
+        },
+        "description": "The collection of software updates found on the computer."
+      },
+      "SoftwareUpdate": {
+        "properties": {
+          "properties": {
+            "x-ms-client-flatten": true,
+            "$ref": "#/definitions/SoftwareUpdateProperties",
+            "description": "Properties of the software update."
+          }
+        },
+        "allOf": [
+          {
+          "$ref": "../../common/v1/definitions.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Describes a specific software update on the given virtual machine."
+      },
+      "SoftwareUpdateProperties": {
+        "description": "Properties of the software update object.",
+        "properties": {
+          "updateState": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Needed / Installed"
+          },
+          "availableOnComputerSince": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Point in time when this update was first seen on this computer"
+          },
+          "approved": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Approval status"
+          },
+          "packageRepository": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Package repository"
+          },
+          "supercededUpdateIds": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Superseded Update Ids"
+          },
+          "isSupercededUpdate": {
```

Comment:
this is modeled as a string but looks like you wanted a bool. If yes, same comment as above about bools. string enum will work better.

Reply:
Done